org.kde.Kid3.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.kde.Kid3
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: kid3
+
+rename-desktop-file: org.kde.kid3.desktop
+rename-appdata-file: org.kde.kid3.appdata.xml
+rename-icon: kid3-qt
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=session-bus
+  - --share=ipc
+  - --device=dri
+
+  # Required to load music metadata
+  - --share=network
+
+  # Required for input-output
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --env=TMPDIR=/var/tmp
+
+  # For MPRIS
+  - --own-name=org.mpris.MediaPlayer2.kid3
+
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+
+modules:
+- name: chromaprint
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  sources:
+    - type: archive
+      url: https://github.com/acoustid/chromaprint/archive/v1.5.0.tar.gz
+      sha256: 5c8e0d579cb3478900699110aa961c1552a422a18741cf67dd62136b1b877c7b
+
+- name: taglib
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DBUILD_SHARED_LIBS=ON
+  - -DWITH_MP4=ON
+  - -DWITH_ASF=ON
+  sources:
+    - type: archive
+      url: https://taglib.org/releases/taglib-1.11.1.tar.gz
+      sha256: b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b
+
+- name: kid3
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DWITH_APPS=KDE;Qt;CLI
+  - -DWITH_ID3LIB=OFF
+  - -DDOCBOOK_XSL_DIR=docbook-xsl
+  sources:
+    - type: archive
+      url: https://invent.kde.org/kde/kid3/-/archive/v3.8.3/kid3-v3.8.3.tar.gz
```

Comment:
I suspect the KDE infra team would rather we use https://download.kde.org/stable/kid3/3.8.3/kid3-3.8.3.tar.xz but I could be wrong ;-)

Reply:
Changed.